source/localizable/addons-and-dependencies/managing-dependencies.md

```diff
@@ -109,7 +111,11 @@ All style assets added this way will be concatenated and output as `/assets/vend
 
 ### Other Assets
 
-All other assets like images or fonts can also be added via `import()`.
+All assets located in the `public` folder will be copied as is to the `dist/` folder.
+
+For example, a `favicon` located at `public/images/favicon.ico` will be copied to `dist/images/favicon.ico`.
+
+All other third-party assets like images or fonts can also be added via `import()`.
```

Comment:
This reads strangely to me because it says "all other third-party assets" comes after a discussion of assets that are explicitly _not_ third party. What does the other mean in this case?

It's not clear here whether this documentation is about files in `vendor/` or `public/`. Files in `public/` are included automatically, as you note. Files in `vendor/` **must** be `app.import()`ed. They won't work at all if you skip this step. We may want to state this in stronger language, and make it clear which rules apply to which directories.

TLDR:

1. Your own, non-JavaScript assets go in `public/` and are copied to the final output directory automatically. You should not (and indeed, cannot) `app.import()` anything in `public/`.
2. Third-party assets, _including_ JavaScript, go inside the `vendor/` directory. In order for a file to be included in the final build output, it _must_ be `app.import()`ed.